packages/cfd/src/features/Containers/mt5-trade-modal.tsx

```diff
@@ -18,76 +33,228 @@ type TMT5TradeModalProps = {
+                        </Text>
+                    </a>
+                </div>
+                <div className='cfd-trade-modal__download-center-app--option cfd-trade-modal__download-center-app--option-hide'>
+                    <Icon icon='IcLinuxLogo' size={32} />
+                    <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                        {localize('MetaTrader 5 Linux app')}
+                    </Text>
+                    <a
+                        className='dc-btn cfd-trade-modal__download-center-app--option-link'
+                        type='button'
+                        href={getPlatformMt5DownloadLink('linux')}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                    >
+                        <Text size='xxs' weight='bold' color='prominent'>
+                            {localize('Learn more')}
+                        </Text>
+                    </a>
+                </div>
+            </div>
+            <Text
+                align='center'
+                as='div'
+                className='cfd-trade-modal__download-center-text'
+                size={isMobile() ? 'xxxs' : 'xxs'}
+                weight='bold'
+            >
+                {localize(
+                    'Download {{ platform }} on your phone to trade with the {{ platform }} {{ account }} account',
+                    {
+                        platform: getCFDPlatformLabel(CFD_PLATFORMS.MT5),
+                        account: getAccountTitle(),
+                    }
+                )}
+            </Text>
+            <div className='cfd-trade-modal__download-center-options'>
+                <div className='cfd-trade-modal__download-center-options--mobile-links'>
+                    <a href={getPlatformMt5DownloadLink('ios')} target='_blank' rel='noopener noreferrer'>
+                        <Icon icon='IcInstallationApple' width={135} height={40} />
+                    </a>
+                    <a href={getPlatformMt5DownloadLink('android')} target='_blank' rel='noopener noreferrer'>
+                        <Icon icon='IcInstallationGoogle' width={135} height={40} />
+                    </a>
+                    <a href={getPlatformMt5DownloadLink('huawei')} target='_blank' rel='noopener noreferrer'>
+                        <Icon icon='IcInstallationHuawei' width={135} height={40} />
+                    </a>
+                </div>
+
+                <div className='cfd-trade-modal__download-center-options--qrcode cfd-trade-modal__download-center-options--qrcode-hide'>
+                    <img src={getUrlBase('/public/images/common/mt5_download.png')} width={80} height={80} />
+                    <Text align='center' size='xxs'>
+                        {localize('Scan the QR code to download {{ platform }}.', {
+                            platform: getPlatformSettings('mt5').name,
+                        })}
```

Comment:
Does the translation for the dynamic string happen correctly? isn't it better to use Localize component!

Reply:
We did not have any issues regarding the translation. I will make these kind of changes while I start working on the refactoring of trade modal and remove its store dependencies.